src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1614,6 +1721,28 @@ void RaftReplDev::complete_replace_member(repl_req_ptr_t rreq) {
     RD_LOGI(rreq->traceID(), "Raft repl replace_member_task has been cleared.");
 }
 
+void RaftReplDev::remove_member(repl_req_ptr_t rreq) {
+    auto member = *r_cast< const replica_id_t* >(rreq->header().cbytes());
+    RD_LOGI(rreq->traceID(), "Raft repl remove_member commit, member={}", boost::uuids::to_string(member));
+    m_listener->on_remove_member(member, rreq->traceID());
+}
+
+void RaftReplDev::clean_replace_member_task(repl_req_ptr_t rreq) {
+    auto task_id = std::string(r_cast< const char* >(rreq->header().cbytes()));
```

Comment:
one stupid question , why not use uint64 instead of string for task_id. that is more simple to persist.

to keep align with cm proto?